src/Entity/Phone.php

```diff
@@ -0,0 +1,550 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\PhoneRepository;
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Entity(repositoryClass=PhoneRepository::class)
+ */
+class Phone
```

Comment:
Codacy found an issue: [The class Phone has 17 fields. Consider redesigning Phone to keep the number of fields under 15.](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6297993)